File: Java/Test6Geo/src/com/company/Punkt.java
package com.company;

public class Punkt {
    public double X;
    public double Y;
    public double Z;

    public Punkt(double x, double y, double z) {
        X = x;
        Y = y;
        Z = z;
    }

    @Override
    public String toString() {
        return "Punkt{" +
                "X=" + X +
                ", Y=" + Y +
                ", Z=" + Z +
                '}';
    }
}

File: C#/5EHIFTroestlStreckenundPunkte/5EHIFTroestlStreckenundPunkte/Punkt.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text.Json;
using System.Threading.Tasks;

namespace _5EHIFTroestlStreckenundPunkte
{
    internal class Punkt
    {
        #region Properties
        public int X { get; private set; }
        public int Y { get; private set; }
        #endregion

        #region Initialize
        public Punkt(int x, int y)
        {
            X = x;
            Y = y;
        }
        #endregion

        #region Overrides
        public override string ToString()
        {
            return JsonSerializer.Serialize(this);
        }
        #endregion
    }
}
